src/components/alert/BasicAlert.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <el-alert :title="title" :type="type"> </el-alert>
+  <el-alert
```

Comment:
We can write it as <el-alert /> as it is single tag. Just for some readability..

Reply:
ok thanks for pointing these things out! I'm still new so I definitely want to improve readability. PS. I started following your way of indenting for props and classes and its much easier to read now!